docs/smart-contracts/plu-ts.md

```diff
@@ -14,7 +14,7 @@ The tool is composed by the eDSL that allows you to create smart contracts and c
 
 ## Getting started
 
-Visit the [Get started with plu-ts](/docs/get-started/plu-ts) to install plu-ts unsing `npm` and setup your project.
+Visit the [Get started with plu-ts](/docs/get-started/plu-ts) to install plu-ts using `npm` and setup your project.
```

Comment:
```suggestion
Visit the [Get started with plu-ts](/docs/get-started/plu-ts) to install `plu-ts` using `npm` and set up your project.
```
... like this error (plus a breach of formatting convention) on the same line, for instance? 🤔